contents/handbook/product/user-feedback.md

```diff
@@ -27,17 +27,22 @@ Ways to invite users for an interview:
 - Add all feedback calls to the [User interviews calendar](https://calendar.google.com/calendar/?cid=Y19tczllaWN1Ym92ZGgxYWhzNmtoY2xpNTQ3b0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t). If the invite was created from your own calendar, you can simply add "User interviews" as an invitee.
 
 **During the call**
-- If you'd like to record the call, do ask before!
+- We recommend recording interviews using [BuildBetter](https://app.buildbetter.app/). Please, always ask and make sure the user is comfortable with being recorded before doing so.
+   - Once the user confirms they are okay with this, you can easily trigger the recording by clicking on `Join a call` on the BuildBetter homepage and copy-pasting the call URL.
+   - If you do not have access to BuildBetter yet, drop a message in the `#pm` Slack channel.
+   - If you have any feedback, bug to report or feature request for BuildBetter, we have a [shared Slack channel](https://posthog.slack.com/archives/C080Q451P3M) with them, feel free to directly message their team there.
 - Do a quick round of intros if you haven't met previously.
 - If this is the first interview with the user, ask them for context about their company, their role, if they're technical.
 
 **After the call**
-1. Add the notes to the [Google Doc][feedback-doc], linking the recording if you did one.
-2. Share a short summary of the user interview in the `#posthog-feedback` Slack channel.
-3. If the user reported specific bugs or requested specific features, open the relevant issues in GitHub. Be sure to link to their person profile in case our engineers needs more context when scoping/building.
-3. Generate the reward for the user (see below).
+1. If you used BuildBetter, the tool will automatically generate a summary for you under the recording. We recommend checking this, and adding any additional thoughts, or you can generate a doc using the platform, where you can give very specific prompts for the outline of the summay.
```

Comment:
```suggestion
1. If you used BuildBetter, the tool will automatically generate a summary for you under the recording. We recommend checking this, and adding any additional thoughts, because the AI can sometimes pick up things incorrectly. You can also generate a doc using the platform, where you can give very specific prompts for the outline of the summary.
```